pkg/kp/rcstore/consul_store.go

```diff
@@ -10,13 +10,20 @@ import (
 	"github.com/square/p2/Godeps/_workspace/src/github.com/hashicorp/consul/api"
 	"github.com/square/p2/Godeps/_workspace/src/github.com/pborman/uuid"
 
+	"github.com/square/p2/pkg/kp"
 	"github.com/square/p2/pkg/labels"
 	"github.com/square/p2/pkg/logging"
 	"github.com/square/p2/pkg/pods"
 	"github.com/square/p2/pkg/rc/fields"
 )
 
-const kvPrefix = "replication_controllers"
+const (
+	RC_TREE string = "replication_controller"
+)
+
+func RcPath(args ...string) string {
```

Comment:
nit: golang typically uses all-caps with abbreviations, and we use "RC" elsewhere too.  Can this be `RCPath()`?